src/ray/protobuf/gcs_service.proto

```diff
@@ -189,6 +189,15 @@ message ReportHeartbeatReply {
   GcsStatus status = 1;
 }
 
+message CheckAliveRequest {
+  int32 seq = 1;
+}
+
+message CheckAliveReply {
+  GcsStatus status = 1;
+  int32 seq = 2;
```

Comment:
Is `status` ever used?

Reply:
Yes, it is used here: https://github.com/ray-project/ray/blob/19026f337eab2716b3a9e520e748f7f5458884ff/dashboard/head.py#L60
And, the status field follows the GCS rpc reply structure.